mods/d2k/rules/harkonnen.yaml

```diff
@@ -1,248 +0,0 @@
-CONYARDH:
-	Inherits: ^CONYARD
-
-PWRH:
-	Inherits: ^POWER
-	Buildable:
-		Prerequisites: ~conyardh
-
-WALLH:
-	Inherits: ^WALL
-	Buildable:
-		Prerequisites: ~conyardh, barracks
-
-GUNTOWERH:
-	Inherits: ^GUNTOWER
-	Buildable:
-		Prerequisites: ~conyardh, barracks
-
-ROCKETTOWERH:
-	Inherits: ^ROCKETTOWER
-	Buildable:
-		Prerequisites: ~conyardh, radar
-
-REFH:
-	Inherits: ^REFINERY
-	Buildable:
-		Prerequisites: ~conyardh, power
-
-BARRH:
-	Inherits: ^BARRACKS
-	Buildable:
-		Prerequisites: ~conyardh, power
-
-REPAIRH:
-	Inherits: ^REPAIR
-	Buildable:
-		Prerequisites: ~conyardh, heavy
-
-RESEARCHH:
-	Inherits: ^RESEARCH
-	Buildable:
-		Prerequisites: ~conyardh, hightech
-
-SILOH:
-	Inherits: ^SILO
-	Buildable:
-		Prerequisites: ~conyardh, refinery
-
-LIGHTH:
-	Inherits: ^LIGHT
-	Buildable:
-		Prerequisites: ~conyardh, refinery
-	ProvidesCustomPrerequisite@TRIKES:
-		Prerequisite: trikes
-	ProvidesCustomPrerequisite@LIGHT:
-		Prerequisite: light
-
-HEAVYH:
-	Inherits: ^HEAVY
-	Buildable:
-		Prerequisites: ~conyardh, refinery
-
-RADARH:
-	Inherits: ^RADAR
-	Buildable:
-		Prerequisites: ~conyardh, barracks
-
-STARPORTH:
-	Inherits: ^STARPORT
-	Buildable:
-		Prerequisites: ~conyardh, radar
-
-HIGHTECHH:
-	Inherits: ^HIGHTECH
-	Buildable:
-		Prerequisites: ~conyardh, radar
-
-PALACEH:
```

Comment:
I guess this removal will make it impossible to setup the launch overlay:

![image](https://cloud.githubusercontent.com/assets/756669/6203029/ca399dea-b50a-11e4-9c4f-0ee012555c17.png)